include/scrimmage/plugins/autonomy/ArduPilot/ArduPilot.xml

```diff
@@ -6,9 +6,12 @@
   <!-- Transmit Socket: State information from SCRIMMAGE to ArduPilot -->
   <to_ardupilot_ip>127.0.0.1</to_ardupilot_ip>
   <to_ardupilot_port>5003</to_ardupilot_port>
+  <from_ardupilot_port>5002</from_ardupilot_port>
 
   <!-- Receive Socket: PWM commands from ArduPilot -->
-  <from_ardupilot_port>5002</from_ardupilot_port>
+<!--   <to_ardupilot_port>5501</to_ardupilot_port>
```

Comment:
Can you remove commented out xml if it is not needed? If it's used in a specific mode, maybe add a comment here.